common/src/Microsoft.Azure.IIoT.Core/src/Hub/Runtime/IoTHubEventConfig.cs

```diff
@@ -45,10 +47,19 @@ public class IoTHubEventConfig : ConfigBase, IEventHubConsumerConfig {
             }
         }
 
-        /// <summary> Event hub consumer group </summary>
+        /// <summary> Event hub default consumer group </summary>
         public string ConsumerGroup => GetStringOrDefault(kEventHubConsumerGroupKey,
             GetStringOrDefault("PCS_IOTHUB_EVENTHUBCONSUMERGROUP",
                 GetStringOrDefault("PCS_IOTHUBREACT_HUB_CONSUMERGROUP", "$default")));
+
+        /// <summary> Event hub telemetry consumer group </summary>
+        public string ConsumerGroupTelemetry => GetStringOrDefault(kEventHubConsumerGroupTelemetryKey,
+            GetStringOrDefault(PcsVariable.PCS_IOTHUB_EVENTHUB_CONSUMER_GROUP_TELEMETRY, "telemetry"));
+
+        /// <summary> Event hub events consumer group </summary>
+        public string ConsumerGroupEvents => GetStringOrDefault(kEventHubConsumerGroupEventsKey,
```

Comment:
Let's put this into the services Config class themselves, not into the interface.  Should be just consumer group, and then in the Processor service you overwrite it with the GetStringOrDefault(kEventHubConsumerGroupTelemetryKey,
            GetStringOrDefault(PcsVariable.PCS_IOTHUB_EVENTHUB_CONSUMER_GROUP_TELEMETRY, "telemetry")); 
etc.

Reply:
Technically these are properties of the IoTHub Event, respectively the EventHub, that the platform requires being created/configured by the user upfront in the context of configuring the resources. By having all the properties of the resource in one place we make the configuration a bit easier.

Reply:
Imo they are specific to how the service requires the it not generically applicable to all.  We would add more and more for every service we add.

Reply:
done